packages/app/src/specs/banners/index.ts

```diff
@@ -0,0 +1,10 @@
+export const BannerIds = {
+  ACI_082022_LOGIN: 'aci_082022_login',
+  ACI_082022_CREATE_ORG: 'aci_082022_createOrganization',
+  ACI_082022_CONNECT_PROJECT: 'aci_082022_connectProject',
+  ACI_082022_RECORD: 'aci_082022_record',
+}
+
+export { default as LoginBanner } from './LoginBanner.vue'
+
+export { default as CreateOrganizationBanner } from './CreateOrganizationBanner.vue'
```

Comment:
I also really like this pattern where we have one directory of something (eg component, concern, whatever) and mark the "public" API by an `index.ts` that exports things. I have been [trying it in my projects](https://github.com/lmiller1990/rhythm/tree/main/packages/frontend/src/screens/gameplay/components/Gameplay) and it's going well... I think we do want to start to explore more rigid architectural patterns (including source files and directories) as we grow the project.

Any thoughts on this pattern?

Reply:
Yes! I started building a barrel file here then apparently got lost halfway through. If nothing else, they really help shorten up import statements. Will finish this one and fix the imports